cmd/blockchaincmd/import_public.go

```diff
@@ -70,6 +71,7 @@ plag.`,
 		"the blockchain ID",
 	)
 	cmd.Flags().StringVar(&rpcURL, "rpc", "", "rpc endpoint for the blockchain")
+	cmd.Flags().BoolVar(&noRPCAvailable, "no-rpc-available", false, "use this when an RPC if offline and can't be accesed")
```

Comment:
do we want to check if user sets invalid option combination here, i.e. `no-rpc-available` as true, but non empty `rpc`